### PR TITLE
Am 2714 st cic wa mapping rules judicial

### PR DESCRIPTION
**Pull request by abul-syed**
https://tools.hmcts.net/jira/browse/AM-2714
ST_CIC_WA - Add mapping rules in ORM for ST judges

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V20230512_2714__AM-2714_SPECIAL_TRIBUNALS_WA_1_0_base_flag_addition.sql
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/main/resources/db/migration/V20230512_2714__AM-2714_SPECIAL_TRIBUNALS_WA_1_0_base_flag_addition.sql:1]: New naming convention is how Matt would like it done going forward

File: src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/model/enums/FeatureFlagEnum.java [hunk 1]
```diff
@@ -10,7 +10,8 @@ public enum FeatureFlagEnum {
     PUBLICLAW_WA_1_0("publiclaw_wa_1_0"),
     IAC_WA_1_2("iac_wa_1_2"),
     CIVIL_WA_1_1("civil_wa_1_1"),
-    PRIVATELAW_WA_1_1("privatelaw_wa_1_1");
+    PRIVATELAW_WA_1_1("privatelaw_wa_1_1"),
+    SPECIALTRIBUNALS_WA_1_0("specialtribunals_wa_1_0");
```
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/model/enums/FeatureFlagEnum.java]: changes made throughout

**Review by mikebrownccd — APPROVED** [review]
LGTM after slight changes that have been made

**Review by mattnayler — COMMENTED** [review]
Reviewed alongside #1472 and current documented mappings in config.

**Pull request merged**